modules/form-field/react/stories/stories_Switch.tsx

```diff
@@ -90,3 +107,42 @@ storiesOf('Components|Inputs/Switch/React/Left Label', module)
       {control(<Switch />)}
     </FormField>
   ));
+
+storiesOf('Components|Inputs/Switch/React/Visual Testing', module)
```

Comment:
"Visual Testing" seems to make sense for me or I could remove the grouping name and organize "States" under `Inputs/Switch/React` instead. Thoughts?

Reply:
I kind of like a visual testing group. It might be too noisy to have every state of complex components represented in one story.